### PR TITLE
#35: Inspection. GraphQL resolver should implement Resolver interface

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will add the code inspection for the graphql resolver class must implement `Magento\Framework\GraphQl\Query\ResolverInterface`
### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: src/com/magento/idea/magento2plugin/php/linemarker/GraphQlResolverUsageLineMarkerProvider.java [hunk 1]
```diff
@@ -57,41 +56,4 @@ public void collectSlowLineMarkers(@NotNull List<PsiElement> psiElements, @NotNu
             }
         }
     }
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/php/linemarker/GraphQlResolverUsageLineMarkerProvider.java:58]: Great refactoring. Love it!

**Comment by VitaliyBoyko** [comment]
@konarshankar07 
Would you like to cover this one by a test?
I've added an example for inspection.
`com.magento.idea.magento2plugin.inspections.xml.CacheableFalseInDefaultLayoutInspectionTest`
Please take a look and let me know whether you want to do this. Thank you!

**Comment by konarshankar07** [comment]
Hello @VitaliyBoyko ....
I will cover this inspection with tests
Thanks

**Comment by VitaliyBoyko** [comment]
Hi @konarshankar07 
You miss understood the task a little bit :)
Inspection should work in the scope of the schema.graphqls
image

However, this one is also useful. So I'm merging it. But the task remains open.

Thank you!

**Pull request merged**